### DISCUSSION TITLE
Current status of the BP project

### DISCUSSION THREAD

**Discussion by filipencopav** [blockprotocol/blockprotocol#1416] [Q&A] [answered]
Hello, is this project still alive? Why is most activity abruptly ending in 2023?

**Comment by vilkinsons** [comment, marked as answer]
Hi @filipencopav, thanks for your question. Version 0.4 of the Block Protocol remains under development, although the vast majority of related experimentation and planning is taking place over in the [hashintel/hash](https://github.com/hashintel/hash) repository.
